src/libraries/System.DirectoryServices/src/Interop/SafeNativeMethods.cs

```diff
@@ -25,10 +25,10 @@ public const int
             ERROR_MORE_DATA = 234,
             ERROR_SUCCESS = 0;
 
-        [DllImport(ExternDll.Activeds, CharSet = CharSet.Unicode)]
+        [DllImport(global::Interop.Libraries.Activeds, CharSet = CharSet.Unicode)]
         public static extern unsafe int ADsGetLastError(out int error, char* errorBuffer, int errorBufferLength, char* nameBuffer, int nameBufferLength);
 
-        [DllImport(ExternDll.Activeds, CharSet = CharSet.Unicode)]
+        [DllImport(global::Interop.Libraries.Activeds, CharSet = CharSet.Unicode)]
```

Comment:
Why are all these `global::`s necessary?  Is there some kind of naming conflict this is working around?

Reply:
> Is there some kind of naming conflict this is working around?

Yes, there is an `Interop` namespace in this project.

Reply:
Ah, I see, the `System.DirectoryServices.Interop` namespace.  Presumably once we clean up the rest of the interop we can remove the "global::".